### PR TITLE
docs(agw): Document Sentry client filters

### PR DESCRIPTION
**Pull request by sebathomas**
## Summary

Add documentation for client filters introduced in #10276.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by alexzurbonsen — COMMENTED** [review]
As far as I can see, the `/networks/{network_id}/gateways/{gateway_id}` endpoint currently doesn't allow the sentry configs to be set. The schema 'network_sentry_config' is currently only used in the other two endpoints.
We might want to adapt the existing docs as well.

**Comment by sebathomas** [comment]
@alexzurbonsen Good catch, changed the existing documentation too.

**Review by themarwhal — APPROVED** [review]

File: docs/readmes/lte/configure_sentry.md [hunk 1]
```diff
@@ -44,6 +44,28 @@ sentry_upload_mme_log: false
 sentry_sample_rate: 1.0
 ```
 
+## Client-side filters for error messages
+
+Another configuration option is a list of error messages that should be filtered and not be sent to Sentry. This is done to avoid spamming Sentry with errors that can occur frequently but that are not bugs to be monitored.
```
Comment (themarwhal) [inline docs/readmes/lte/configure_sentry.md:49]: Maybe we can say that this is similar in concept to the server side fliter configuration made available by Sentry.io: https://docs.sentry.io/product/accounts/quotas/#inbound-data-filters

Comment (sebathomas) [reply docs/readmes/lte/configure_sentry.md:49]: Added some information about the server-side filters.

**Comment by Neudrino** [comment]
Affected by #11350.

**Pull request merged**